Add private build plan for Ioslabka variant.
--- /dev/null
+++ b/private-build-plans.toml
@@ -0,0 +1,33 @@
+# We aim to leave everything near default slab/fixed, but override stylistic
+# choices (generally going for serifs) for all upright/italic/oblique at once.
+
+[buildPlans.ioslabka]
+
+family = "Ioslabka"
+
+design = [ "v-a-doublestorey"   # cv01
+         , "v-f-serified"       # cv84
+         , "v-i-serified"       # cv03
+         , "v-k-cursive"        # cv70
+         , "v-l-serified"       # cv07
+         , "v-r-serified"       # cv86
+         , "v-t-flat-hook"      # VXDC
+         , "v-u-with-bar"       # cv89
+         , "v-y-cursive"        # cv49
+
+         , "v-capital-g-toothless"          # cv92
+         , "v-capital-j-serifed-both-sides" # VXDA
+         , "v-capital-m-flat-bottom"        # VXCJ
+
+         , "v-zero-dotted"          # cv14
+         , "v-one-base"             # cv51
+         , "v-three-flattop"        # cv46
+         , "v-six-open-contour"     # VXAF
+         , "v-nine-open-contour"    # cv97
+
+         , "v-paren-larger-contour" # VXAO
+         , "v-dollar-open"          # cv38
+
+         , "sp-fixed"
+         , "slab"
+         ]
